src/signals/incident-management/containers/StandardTextsAdmin/components/OverviewPage/OverviewPage.tsx

```diff
@@ -109,7 +110,12 @@ export const OverviewPage = () => {
             setStatusFilter={setStatusFilter}
             setActiveFilter={setActiveFilter}
           />
-          <Button variant="secondary">Tekst toevoegen</Button>
+          <Button
+            variant="secondary"
+            onClick={() => handleOnClick(routes.standardTextsDetailNew)}
```

Comment:
just navigate from here in the anonymous function without all the boilerplate

Reply:
Done